sympy/stats/error_prop.py

```diff
@@ -0,0 +1,74 @@
+"""Tools for arithmetic error propogation."""
+from __future__ import print_function, division
+from itertools import repeat
+
+from sympy import Symbol, symbols, Add, Mul, simplify, Pow, exp
```

Comment:
@certik maybe not exactly what you ask for, but still worth mentioning: https://confluence.jetbrains.com/display/PYH/Configuring+PyCharm+to+work+as+a+Vim+editor